static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -180,7 +180,7 @@ aggregate_rel_date_filter
 
 // has filter for not null type checks
 has_filter
-  = negation:negation? &"has:" key:search_key sep value:(search_key / search_value) &{
+  = negation:negation? &"has:" key:search_key sep value:(search_value/search_key) &{
```

Comment:
@evanpurkhiser I swapped this as the filter.value.type was being reported as keySimple instead of valueText. Not sure if this was intended or a bug (it didnt even seem to be possible given the typedef of Token.Filter)

Reply:
@malwilley you've been in here more recently than me, wdyt

Reply:
Seems like a good change to me

Reply:
Thanks for taking a look @malwilley @evanpurkhiser 🙏🏼